### PR TITLE
added Huntington Beach

### PR DESCRIPTION
**Pull request by codeswitching**

### CODE REVIEW

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    service_charge:
+      depends_on: meter_size
+      values:
+         3/4"       : 11.531
+         1"          : 23.062
+         1_1/2"        : 34.593
+         2"          : 57.655
+         3"          : 126.841
```
Comment (vincent-hebble) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: compound and FM are meter_types

it should be formatted

depends_on:
  - meter_size
  - meter_type
values:
  3/4"|FM               : 11.531
  3/4"|compound  : 11.531

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  # applies to RESIDENTIAL_SINGLE, RESIDENTIAL_MULTI, COMMERCIAL, and INDUSTRIAL but unclear. Cannot find any other rates
+  
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    service_charge:
+      depends_on: meter_size
+      values:
+         3/4"       : 11.531
+         1"          : 23.062
+         1_1/2"        : 34.593
```
Comment (vincent-hebble) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: should be 1 1/2":

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (vincent-hebble) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: Also the Utility ID is 277

the directory should be named "Huntington Beach City of - 277"

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs (hunk 2 above)
Comment (codeswitching) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: Ok, I will update. I did see some older example files that used 1_1/2.

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs (hunk 3 above)
Comment (codeswitching) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: fixed

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs (hunk 2 above)
Comment (codeswitching) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: fixed

File: full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs (hunk 1 above)
Comment (codeswitching) [inline full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs]: fixed

**Comment by codeswitching** [comment]
All issues resolved and new file pushed

**Comment by vincent-hebble** [comment]
Thank You!

**Pull request merged**